abba/json/include/Parse.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef PARSE_HPP
+#define PARSE_HPP
+
+#include "Json.hpp"
+#include <stdlib.h>
+
+class Parse
+{
+private:
+	std::fstream	*_file;
+	size_t			_prevPos;
+	Json			*_root;
+	Json			*_current;
+	Json			json;
+
+private:
+	enum jsonType
```

Comment:
The name `jsonType` is a bit confusing to me, something such as `parseState` would be more accurate considering you also have states for individual characters